src/main/java/Darartole/exception/EmptyBotException.java

```diff
@@ -0,0 +1,15 @@
+package Darartole.exception;
+
+/**
+ * Deals with all the possible exception
+ */
+public class EmptyBotException extends Exception {
+    /**
+     * Throws exception when the bot receives the empty input 
+     * 
+     * @param message the message that is printed out when the exception is thrown
```

Comment:
Hello super minor nitpick here: there is a short sentence in the Java Coding Standard that requires punctuation behind each parameter description.
```suggestion
     * @param message the message that is printed out when the exception is thrown.
```